.eslintignore

```diff
@@ -4,3 +4,5 @@ node_modules
 dist
 # don't lint nyc coverage output
 coverage
+# don't lint npm-specific scripts
+scripts
```

Comment:
was having issues with eslint related to tsconfig not detecting the scripts directory or something like that so opted to just ignore for now. happy to change if someone knows how to properly ignore via config